app/components/email_alert_fieldset_component.rb

```diff
@@ -55,4 +39,30 @@ def render_external_condition_inputs
       value: @email_alert.link,
     })
   end
+
+private
+
+  def email_topic_list_title_prefix(input_name)
```

Comment:
Would it make sense to prefix this method name (and the other one) with "render"?